dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.mpCompatible-6.0.feature

```diff
@@ -0,0 +1,8 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
This feature file and the mpConfig ones should be removed and then you update to get the latest code in release or integration.